Makefile

```diff
@@ -74,6 +74,23 @@ ifeq (, $(SHELLCHECK_COMMAND))
 endif
 SHELL_FILES := $(shell find $(MAKEFILE_DIR) -type f -name '*.sh')
 
+.PHONY: lint
+transform-al2-to-al2023-config:
+	cat $(PACKER_TEMPLATE_FILE) | \
+	jq '._comment = "All template variables are enumerated here; and most variables have a default value defined in eks-worker-al2023-variables.json"' | \
+	jq '.variables.temporary_key_pair_type = "ed25519"' \
+	> $(subst al2,al2023,$(PACKER_TEMPLATE_FILE))
+
+	# TODO: switch to minimal "al2023-ami-minimal-2023.*-kernel-6.1-x86_64"
+	cat $(PACKER_DEFAULT_VARIABLE_FILE) | \
+	jq '.ami_component_description = "(k8s: {{ user `kubernetes_version` }}, containerd: {{ user `containerd_version` }})"' | \
+	jq '.ami_description = "EKS-optimized Kubernetes node based on Amazon Linux 2023"' | \
+	jq '.containerd_version = "*" | .runc_version = "*" | .docker_version = "" ' | \
+	jq '.source_ami_filter_name = "al2023-ami-2023.*-kernel-6.1-x86_64"' | \
+	jq '.launch_block_device_mappings_volume_size = "8"' | \
+	jq '.volume_type = "gp3"' \
+	> $(subst al2,al2023,$(PACKER_DEFAULT_VARIABLE_FILE))
+
```

Comment:
Can we throw this logic into a script in `hack/` to keep things a bit cleaner? (and the bash can look nicer there)

Reply:
Done